README.md

```diff
@@ -1,5 +1,6 @@
 # Prometheus Go client library
 
+[![GoDoc](http://img.shields.io/badge/go-documentation-blue.svg?style=flat-square)](http://godoc.org/github.com/prometheus/client_golang)
```

Comment:
Could you use the same style and order as for the GoDoc badges further down? (Either make it here the same as below or change below to this style, if you like it more.)

Reply:
Ah, I didn't realise they were there. No need for this PR then.